api/app/src/main/kotlin/packit/controllers/PacketController.kt

```diff
@@ -70,4 +72,15 @@ class PacketController(private val packetService: PacketService)
             .headers(response.second)
             .body(response.first)
     }
+
+    @GetMapping("/packetGroup")
```

Comment:
yeah initially was iffy because packetGroup is like a subset of packet (derived ) but you are right... il do as part of #87 because I've made changes to endpoint